ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -97,6 +100,20 @@ public void setAccessibilityComponentType(T view, String accessibilityComponentT
     AccessibilityHelper.updateAccessibilityComponentType(view, accessibilityComponentType);
   }
 
+  @ReactProp(name = PROP_ACCESSIBILITY_ELEMENTS_HIDDEN)
+  public void setImportantForAccessibility(T view, @Nullable Boolean accessibilityElementsHidden) {
```

Comment:
No reason, I have no idea how I forgot to change that after copying the initial `setImportantForAccessibility` method. Pushing a fix now.